src/App.js

```diff
@@ -13,10 +15,155 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Goods 1</h1>
-  </div>
-);
+class App extends React.Component {
+  state = {
+    originalGoodsList: [...goodsFromServer],
+    goodsList: [...goodsFromServer],
+    isButtonClicked: false,
```

Comment:
кнопок много, о какой именно идет речь?
```suggestion
    isLoaded: false,
```
```suggestion
    isStarted: false,
```

что-то в этом духе